config.py

```diff
@@ -1,33 +1,43 @@
-# Discord API / Login settings
-TOKEN = "yo_momma's_sized_token_here"
+from configparser import ConfigParser
 
+config = ConfigParser()
+
+with open('config.ini') as fp:
+    config.read_file(fp)
+
+discord = config['discord']
+
+TOKEN = discord['token']
 
 # Rules 'n' settings
-max_channels_per_category = 50
-max_participants = 40
-max_cc_per_user = 8
-season = "1"
+max_channels_per_category = int(discord['max_channels_per_category'])
+max_participants = int(discord['max_participants'])
+max_cc_per_user = int(discord['max_cc_per_user'])
+season = discord['season']
```

Comment:
How is Discord supposed to know what I want as my maximum amount of participants?

Reply:
excuse me, did you read the code?